test/extended/project/project.go

```diff
@@ -132,14 +132,21 @@ var _ = g.Describe("[Feature:ProjectAPI] ", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 			waitForAdd(ns01Name, beginningWatch)
 
-			fromNowWatch, err := bobProjectClient.Projects().Watch(metav1.ListOptions{})
-			o.Expect(err).NotTo(o.HaveOccurred())
-			select {
-			case event := <-fromNowWatch.ResultChan():
-				g.Fail(fmt.Sprintf("unexpected event %s %#v", event.Type, event.Object))
+			// TODO: Move to openshift-apiserver as a unit test in a non-HA environment.
```

Comment:
looking at what this is testing, can't we simply sleep a fair amount before starting this watch?